### ISSUE TITLE
VNet az140-adds-vnet1 does not exist

### ISSUE THREAD

**Issue by rramoscabral** [MicrosoftLearning/AZ-140-Configuring-and-Operating-Windows-Virtual-Desktop#1]
# Module: 01
#### Step: 03

Description of issue

The VNet az140-adds-vnet11 defined in ARM deployment  does not exist only az140-aadds-vnet11a was created.

Repro steps:

1. Execute New-AzResourceGroupDeployment `
  -ResourceGroupName $resourceGroupName `
  -Location $location `
  -Name az140lab0101vmDeployment `
  -TemplateFile $HOME/az140-11_azuredeploycl11a.json `
  -TemplateParameterFile $HOME/az140-11_azuredeploycl11a.parameters.json

**Comment by polichtm** [comment]
Hi Ricardo,
updated
rgds
Marcin

**Comment by rramoscabral** [comment]
By erasing everything and doing it all over again I realized that I made a mistake in my explanation and didn't even notice.

The VNet 'VNet az140-adds-vnet11' is created in "LAB_01L01_Prepare_for_deployment_of_WVD_ADDS" on Exercise 1, Task 2, step 4 to 8.

The problem I had was with "LAB_01L01_Prepare_for_deployment_of_WVD_AADDS".

I tried again and both work without any problems.

In Task 3 from "WVD_AADDS "VM": Create AD DS users and groups that will be synchronized to Azure AD DS
  - Step 1: "Within the Remote Desktop to the az140-cl-vm11a Azure VM, start Microsoft Edge, navigate to the Azure portal, and sign in by providing user principal name of the aadmin1 user account with Pa55w.rd1234 as its password."
  - I use aadadmin1.
